components/DynamicContent/src/index.scss

```diff
@@ -52,10 +65,27 @@
   }
 }
 
+.denhaag-dynamic-content__card-image-container {
+  height: var(--denhaag-dynamic-content-image-container-height);
+  width: var(--denhaag-dynamic-content-image-container-width);
+  overflow: hidden;
+}
+
 .denhaag-dynamic-content__card-image {
   aspect-ratio: 16 / 9;
   background: var(--denhaag-dynamic-content-image-background-color);
   object-fit: cover;
+  transition: all var(--denhaag-dynamic-content-card-transition-timing-function, ease-in-out)
```

Comment:
Target specific changes. Now you make the transition heavier for no reason.

Reply:
done